stacks/openshift/application/oidc-authservice-for-appid/params.env

```diff
@@ -0,0 +1,9 @@
+client_id=xxxxxx
+oidc_provider=https://us-east.appid.cloud.ibm.com/oauth/v4/xxxx
+oidc_redirect_uri=https://<kubeflow-gateway-host>/login/oidc
```

Comment:
is it possible to use the internal gateway?

Reply:
No, it must be the external FQDN of the istio ingressgateway. This option will be configured in AppID side and sent to AppID to be validated. since AppID only redirect authenticated requests to valid redirect URL.

Reply:
k, then we should have clear instruction on how to get that hostname and port, in different situations.